extensions/arc/package.json

```diff
@@ -9,7 +9,7 @@
   "icon": "images/extension.png",
   "engines": {
     "vscode": "*",
-    "azdata": ">=1.19.0"
+    "azdata": ">=1.20.0"
```

Comment:
are we relying on any new changes in ADS?

Reply:
Not for the deployment - but Brian made some fixes to the model view stuff so this extension will need the latest ADS version.